src/components/RightSidebar/RightSidebar.vue

```diff
@@ -273,30 +273,33 @@ export default {
 
 	watch: {
 		conversation(newConversation, oldConversation) {
-			if (!this.isRenamingConversation) {
-				this.conversationName = this.conversation.displayName
-			}
+			// Waiting for chat tab to mount / destroy
+			this.$nextTick(() => {
```

Comment:
It is not clean to me, why had we to wait until the chat tab is mounted or destroyed for all this actions?

Reply:
See watchers and methods for https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppSidebar/NcAppSidebarTabs.vue:

If there is no tab provided (before `$nextTick`, for example), library component will switch to the first available tab ('shared items' in our case). So we need to mount 'participants' tab component, and only after that set `:active` to 'participants'

Reply:
The problem is in `NcAppSidebarTabs`, it ignores activating the current tab after registration.

Reply:
Works perfect with mentioned patch in vue-lib!